ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -475,6 +475,7 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskOption) {
                                       localized text is displayed.
  @param distanceInMeters            The timed walk distance in meters.
  @param timeLimit                   The time limit to complete the trials.
+ @param turnAroundTimeLimit         The time limit to complete the turn around step.
```

Comment:
Please *add* a method rather than replacing an existing one. This task module is used by other apps and this is a breaking change.